### DISCUSSION TITLE
SEO Sitemap generates with double slashes

### DISCUSSION THREAD

**Discussion by alXDraghiciu** [OrchardCMS/OrchardCore#12756] [General]
### Describe the bug

Hi. I've enabled the Sitemap module and proceeded to generate one for the website I'm working on. Problem is all the generated URL's have double slashes.

![screenshot](https://i.imgur.com/cThkJJz.png)

The **baseUrl** I've configured under **Settings > General** is:
<br>
`https://mydomain.com`
<br>
**No end slash in the baseUrl**.

Please let me know what could be done to fix this since it seems to have a bad impact on the Google Search Console.

**Comment by Skrypt** [comment]
You need to remove the "/" in your sitemap custom paths. The hint underneath the input shows that it doesn't need a leading slash. Else, I'm keeping this issue opened so that it be validated and removed automatically.

![image](https://user-images.githubusercontent.com/3228637/198353929-17df2560-dcfb-45b1-95e0-f9c6bc436186.png)

![image](https://user-images.githubusercontent.com/3228637/198353963-36de8920-c26a-4fd2-b2a6-7d4c9ce8e367.png)

**Comment by alXDraghiciu** [comment]
I did not have a slash there. I tried to add and remove it but the SiteMap result is always the same. Maybe they are older results? How to you reset the SiteMap to get the latest content?

**Comment by jtkech** [comment]
Check if your items  have a leading slash in their autoroute path.

To invalidate, publish an item or remove the cached files under the app wwwroot folder

As I remember there is an admin action to purge the cache.

**Reply by alXDraghiciu** [reply]
Hello. 

Sorry, you were right. I had an .XML file under the wwwroot and manually deleting it solved the issue. 

It seems that it couldn't be overwritten, so generating new Sitemaps never worked. The old one with an extra slash was in wwwroot and was always served on the URL. Deleting it solved the issue.

One more question: should the Sitemap .XMLs generated in wwwroot by orchard be synced in git? Or should I gitignore them without problems?

**Reply by jtkech** [reply]
You can .gitignore them without any problem ;)

**Reply by alXDraghiciu** [reply]
Oops I meant under `wwwroot/sm-cache/Default/`. Safe to gitignore those?

**Reply by jtkech** [reply]
Yes
